relay-ourlogs/src/ourlog.rs

```diff
@@ -137,20 +151,20 @@ pub fn otel_to_sentry_log(otel_log: OtelLog) -> Result<OurLog, Error> {
 
     let mut other = Object::default();
     other.insert(
-        "severity_text".to_string(),
+        "severity_text".to_owned(),
         Annotated::new(Value::String(severity_text)),
     );
     other.insert(
-        "severity_number".to_string(),
+        "severity_number".to_owned(),
         Annotated::new(Value::I64(severity_number as i64)),
     );
-    other.insert("trace_flags".to_string(), Annotated::new(Value::I64(0)));
+    other.insert("trace_flags".to_owned(), Annotated::new(Value::I64(0)));
     other.insert(
-        "timestamp_nanos".to_string(),
+        "timestamp_nanos".to_owned(),
         Annotated::new(Value::U64(otel_log.time_unix_nano)),
     );
     other.insert(
-        "observed_timestamp_nanos".to_string(),
+        "observed_timestamp_nanos".to_owned(),
         Annotated::new(Value::U64(observed_time_unix_nano)),
     );
```

Comment:
We don't need `other` on the log protocol if you are sending it to the new consumer now, we only had to have this so the `LogKafkaMessage` matched what the consumer was expecting.

We can get rid of these in `ourlog_merge_otel` as well.